lib/kubeclient.rb

```diff
@@ -67,13 +69,26 @@ def all_entities
     private
 
     def validate_auth_options(opts)
+      bearer_token_keys = [:bearer_token, :bearer_token_file]
+      exclusive_keys = bearer_token_keys << :user
```

Comment:
Would this be equal to:

    exclusive_keys = [:bearer_token, :bearer_token_file, :user]

Any reason to define `bearer_token_keys`?